editor/editor_log.cpp

```diff
@@ -252,6 +252,11 @@ void EditorLog::_rebuild_log() {
 }
 
 void EditorLog::_add_log_line(LogMessage &p_message, bool p_replace_previous) {
+	if (!is_inside_tree()) {
+		// The log will be built all at once when it enters the tree and has its theme items.
+		return;
+	}
+
```

Comment:
> I'll try and get this fixed today

Well that was a long day 🙃

So the problem was from https://github.com/godotengine/godot/pull/64271. Adding an error message to the log requires access to its theme items (e.g. error icon). https://github.com/godotengine/godot/blob/060d0ce88204f9180beb3764048975e9dcf5c496/modules/gltf/register_types.cpp#L99 is getting called before the EditorLog has entered the SceneTree. This means `NOTIFICATION_THEME_CHANGED` won't have been called yet, and the EditorLog's theme cache won't have been initialized. Fortunately, the log is already rebuilt anyway  on `NOTIFICATION_THEME_CHANGE`, so I added this check to simply return early if the EditorLog hasn't entered the scene yet. Theoretically this should be more performant anyway since it won't have to add the log line twice.